cpp/include/cudf/transform.hpp

```diff
@@ -111,5 +111,30 @@ std::pair<std::unique_ptr<cudf::column>, std::unique_ptr<cudf::column>> encode(
   cudf::column_view const& input,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Creates a boolean column from given bitmask.
+ *
+ * Each bit represents one value, 0 - false and 1 - true.
+ * It is user's responsibility to make sure the bitmask agrees
+ * with given offset and length.
+ *
+ * Examples:
+ * @code{.pseudo}
+ * input: {147}
```

Comment:
I think it'd be more illustrative to show the corresponding bit pattern. This also reveals that your example has an error. 
```suggestion
 * input: {0b10010011}
 * output: [{true, true, false, false, true, false, false, true}]
```